dandiapi/api/_copy.py

```diff
@@ -13,15 +18,21 @@
 PART_SIZE = 500 * 1024 * 1024  # 500 MB
```

Comment:
@dchiquito From what I can tell the maximum part size is `5GiB`. Should we change this to match that, so that we're not being intentionally inefficient?